### PR TITLE
Convert to python 3 and Serverless framework

### PR DESCRIPTION
**Pull request by boroivanov**

### CODE REVIEW

File: functions/aws_keys_audit.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    access_keys = iam.list_access_keys(UserName=user_name)
+    active_keys = [k for k in access_keys['AccessKeyMetadata']
+                   if k['Status'] == 'Active']
+
+    if len(active_keys) == 0:
+        return {'ComplianceType': 'NOT_APPLICABLE'}
+    elif len(active_keys) > 1:
+        return {
+            'ComplianceType': 'NON_COMPLIANT',
+            'Annotation': 'User has two active keys.'
```
Comment (skout23) [inline functions/aws_keys_audit.py:36]: "User has %d active keys", len(active_keys)?

File: functions/aws_keys_audit.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+def ordering_timestamp(invoking_event):
+    return datetime.strptime(
+        invoking_event['notificationCreationTime'], '%Y-%m-%dT%H:%M:%S.%fZ'
+    )
+
+
+def scheduled_evaluation(invoking_event, result_token):
+    # Periodic evaluation, evaluate all users
+    resp = iam.list_users()
```
Comment (skout23) [inline functions/aws_keys_audit.py:60]: Does this get all users or a limited max? I forget if we need to paginate over the full list.

Comment (boroivanov) [reply functions/aws_keys_audit.py:60]: oh good point. I'll check.

Comment (boroivanov) [reply functions/aws_keys_audit.py]: Done.

**Pull request merged**